### PR TITLE
added a html and css file of home page

### PR DESCRIPTION
**Pull request by Vee-program**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Home</title>
```
Comment (Mavooma) [inline index.html:7]: Detail the Title (e.g. Tech Girls Hub Homepage)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          <li> <img
+              src="logo-image.png"
+              alt="logo image of techgirlshub"
+              class="logo" /></li>
+          <li><a href="#about-Us">About Us</a></li>
+          <li><a href="Services.html">Services</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="#team-members">Team members</a></li>
+           <li><a href="feedbackRating.php">Feedback Rating</a></li>
+          <li><a href="#contact-Us">Contact Us</a></li>
```
Comment (Phile105) [inline index.html:22]: Add a link for contact us as contactUs.php

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        <h2 class="mission-statement">
+              Our mission is to leverage our technical expertise and innovative
+              solutions to empower clients with seamless service requests and
+              communication. We aim to showcase our completed projects and
+              upcoming releases, while providing an interactive platform for
+              visitors to engage with our team members and appreciate their
+              unique contributions.
+            </h2>
+            <div class="btn-container">
+        <a href="#" class="abt-btn">Explore our services</a>
```
Comment (Phile105) [inline index.html:41]: Add a link that takes you to the services page on the button e.g Services.html

File: index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        <a href="#" class="contact-btn">Get in touch</a>
```
Comment (Phile105) [inline index.html:42]: Add the Contact Us link e.g contactUs.php

File: index.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    <footer>
+        <div class="footer-container">
+          <h2>TechGirlsHub</h2>
+          <div class="footer-subtext">TechGirlsHub is a team of women in tech, showcasing projects, 
+            expertise and service while connecting with clients</div>
+          <div class="header-elements">
+          <div class="home">
+            <h2 class="header-home">Home</h2>
+            <a href="#">Home</a><br/>
+            <a href>Services</a><br/>
```
Comment (Phile105) [inline index.html:58]: Link the services page

File: index.html (hunk 2 above)
Comment (techgirlshub) [reply index.html:22]: and about us link

**Review by techgirlshub — CHANGES_REQUESTED** [review]
Please look at the mentioned issues, once fixed click on Resolve conversation so we know it is fixed.

File: style-home.css [hunk 6]
```diff
@@ -0,0 +1,176 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+  background-color: rgb(9, 6, 59);
+  background-image: url("hero-image.jpg");
```
Comment (techgirlshub) [inline style-home.css:9]: it should be   url("images/hero-image.jpg")

File: index.html [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+            <a href="#">Team members</a><br/>
+            <a href>Contact Us</a>
+          </div>
+          </div>
+          <div class="icon-container">
+          <img src="github.png" alt="github icon" class="icon"/>
+          <img src="in.png" alt="linkedin icon" class="icon"/>
+          <img src="instagram.png" alt="instagram icon" class="icon"/>
+          <img src="tiktok.png" alt="twitter icon" class="icon"/>
+        </div>
```
Comment (techgirlshub) [inline index.html:73]: use icons instead of images. check the other pages to see how.

**Pull request closed without merging**